js/stat.js

```diff
@@ -0,0 +1,61 @@
+var CLOUD_WIDTH = 420;
+var CLOUD_HEIGHT = 270;
+var CLOUD_X = 150;
+var CLOUD_Y = 10;
+var TITLE_HEIGHT = 70;
+var GAP = 10;
+var TEXT_HEIGHT = 15;
+var TEXT_WIDTH = 50;
+var BAR_WIDTH = 40;
+var BAR_HEIGHT = CLOUD_HEIGHT - TEXT_HEIGHT - TEXT_HEIGHT - GAP - TITLE_HEIGHT;
+
+var renderCloud = function(ctx, x, y, color) {
+	ctx.fillStyle = color;
+	ctx.fillRect(x, y, CLOUD_WIDTH, CLOUD_HEIGHT);
+};
+
+var getMaxElement = function(arr) {
+	var maxElement = arr[0];
+	
+	for(var i = 0; i < arr.length; i++) {
+		if(arr[i] > maxElement) {
+			maxElement = arr[i];
+		}
+	}
+	
+	return maxElement;
+};
+
+function randomNum(max) {
+	return Math.floor(Math.random() * max);
+};	
+
+window.renderStatistics = function(ctx, names, times) {
+	
+	renderCloud(ctx,  CLOUD_X + GAP, CLOUD_Y + GAP, 'rgba(0, 0, 0, 0.7)');
```

Comment:
```suggestion
	renderCloud(ctx,  CLOUD_X + GAP, CLOUD_Y + GAP, GREY_COLOR);
```
Надо вынести в переменную - магических чисел не должно быть